docs/usage_and_configuration/black_docker_image.md

```diff
@@ -10,6 +10,9 @@ _Black_ images with the following tags are available:
 - `latest_release` - tag created when a new version of _Black_ is released.\
   ℹ Recommended for users who want to use released versions of _Black_. It maps to [the latest release](https://github.com/psf/black/releases/latest)
   of _Black_.
+- `latest_prerelease` - tag created when a new alpha (prerelease) version of _Black_ is
+  released.\
+  ℹ Recommended for users who want to preview or test alpha versions of _Black_.
```

Comment:
One concern here is that we don't usually use prereleases, we're just doing it right now as we're preparing for the 23.1 release. I'm expecting that for the rest of 2023 we won't have any prereleases, then we'll have 24.1a releases again in December 2023. So I think we should at least call out here that there may be a release that's more recent than the latest prerelease.

Reply:
Good call. I did think of this but thought people could deal with it. But being more explicit is only going to help.

Maybe just call it prerelease and state it's only planned generally once a year for testing `--preview` moving into releases.

Reply:
That's good to know, and from a documentation standpoint I think it fits best in [the Release Process docs](https://black.readthedocs.io/en/latest/contributing/release_process.html). Currently there's no mention there of alpha versions or this yearly process, and if there's a thought that this will happen each December, it might be a good addition there.

Then, this bit here can be a bit more specific and possibly also point to the release process docs for more detail.

Thoughts?

Reply:
That makes sense! I kind of came up with this alpha process on the fly because I thought it'd be helpful. However, writing that new documentation would be a bigger change that's outside the scope of this PR, so for now we can just have a brief note here like "Note that the most recent release may be newer than any prerelease."

Reply:
Added, take a gander at that whenever you have a second